external-import/ransomwarelive/src/lib/ransomConn.py

```diff
@@ -478,7 +494,7 @@ def stix_object_generator(self, item, group_data):
                 relation_VI_IS.get("id"),
                 relation_IS_TA.get("id"),
             ],
-            published=datetime.strptime(item.get("published"), "%Y-%m-%d %H:%M:%S.%f"),
+            published=report_published,
             created=datetime.strptime(item.get("discovered"), "%Y-%m-%d %H:%M:%S.%f"),
```

Comment:
```suggestion
            created=datetime.fromisoformat(item.get("discovered")),
```
https://docs.python.org/3.7/library/datetime.html#datetime.datetime.fromisoformat